mediorum/server/replicate.go

```diff
@@ -64,22 +63,6 @@ func (ss *MediorumServer) replicateToMyBucket(fileName string, file io.Reader) e
 		}
 	}
 
-	// record that we "have" this key
```

Comment:
to clarify, are we assuming all the nodes have a full picture of the qm_cids table and nothing else needs to be shared out regarding that? i'm wondering what would happen if a new node registers - does it need to seed qm_cids from somewhere?

Reply:
yeah that's a valid concern... we'll have to add a way to sync that table around (that isn't crudr of course)...
maybe re-add pg_beam stuff for this since it worked so well last time haha!
no but I think that would be a reasonable way if it's quick and easy.